app/binary-intro/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import React, { useState } from 'react';
+import { Typography, Stack, Button, Box } from '@mui/material';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
+import AutorenewIcon from '@mui/icons-material/Autorenew';
+
+const Page = () => {
+  const [decimalNumber, setDecimalNumber] = useState(0);
+
+  const increment = () => {
+    setDecimalNumber(decimalNumber + 1);
+  };
+
+  const decrement = () => {
+    if (decimalNumber > 0) {
+      setDecimalNumber(decimalNumber - 1);
+    }
+  };
+
+  const reset = () => {
+    setDecimalNumber(0);
+  };
+
+  const binaryNumber = decimalNumber.toString(2);
+
+  return (
+    <Box
+      display="flex"
+      justifyContent="center"
+      width="100%"
+    >
+      <Stack direction="column" alignItems="center">
```

Comment:
I think we can replace this with a single Stack component that also has `justifyContent="center"`

Reply:
ok